src/dftbp/dftbplus/parser.F90

```diff
@@ -402,6 +402,8 @@ subroutine readGeometry(node, input)
       call readTGeometryXyz(value1, input%geom)
     case ("vaspformat")
       call readTGeometryVasp(value1, input%geom)
+    case ("lammpsformat")
```

Comment:
This probably needs a regression tester for something simple.

Reply:
Good idea, I will add a test.